src/main/java/org/jabref/gui/groups/GroupDialog.fxml

```diff
@@ -123,7 +123,7 @@
                     </VBox>
                     <VBox visible="${autoRadioButton.selected}" spacing="10.0">
                         <RadioButton fx:id="autoGroupKeywordsOption" toggleGroup="$autoGroupOptions"
-                                     text="Generate groups from keywords in a BibTeX field"/>
+                                     text="Generate groups from keywords in the following field"/>
```

Comment:
This string should be localized as well: add `%` before the text, i.e. `text="%Generate..."` and add it to the JabRef_en file. 
Same for the RadioButton further below.
This can also be done in a follow-up PR as it's not direclty related to the objective of this PR.

Reply:
Localization done!